app/views/hiring_staff/vacancies/edit_vacancy_sections/_edit_job_specification.html.haml

```diff
@@ -1,78 +1,50 @@
-%h2.govuk-heading-m.mb0
+%h2.govuk-heading-m.app-task-list__section#job_details_heading
+  %span.app-task-list__section-number 1.
   = t('jobs.job_details')
+  - if @vacancy.published?
+    = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id), class: 'govuk-link change-button', 'aria-label': t('jobs.aria_labels.change_job_details')
+  - else
+    = link_to t('buttons.change'), job_specification_school_job_path, class: 'govuk-link change-button', 'aria-label': t('jobs.aria_labels.change_job_details')
   - unless @vacancy.job_roles&.any?
     .notification-tag
       %strong.govuk-tag.app-task-list__task-completed
         = t('jobs.notification_labels.new')
 
-.govuk-body-s.mb1
-  = link_to edit_school_job_job_specification_path(@vacancy.id), class: 'govuk-link', 'aria-label': t('jobs.aria_labels.change_job_specification') do
-    Change
-    %span.govuk-visually-hidden the job specification
-
 %dl.app-check-your-answers.app-check-your-answers--short
 
   .app-check-your-answers__contents
-    %dt.app-check-your-answers__question#job_title
-      = t('jobs.job_title')
-    %dd.app-check-your-answers__answer
-      = @vacancy.job_title
-    %dd.app-check-your-answers__change
-      = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'job_title'), 'aria-label': t('jobs.aria_labels.change_job_title'), class: 'govuk-link'
-
-  .app-check-your-answers__contents
-    %dt.app-check-your-answers__question#job_role
-      = t('jobs.job_roles')
-    %dd.app-check-your-answers__answer
-      = @vacancy.show_job_roles
-    %dd.app-check-your-answers__change
-      = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'job_role'), 'aria-label': t('jobs.aria_labels.change_job_role'), class: 'govuk-link'
+    %dt.app-check-your-answers__question.first-question#job_title
```

Comment:
This pattern is repeated a lot of times, you may want to put it in a shared partial